src/components/v5/shared/UserInfoPopover/partials/UserInfo.tsx

```diff
@@ -134,7 +138,14 @@ const UserInfo: FC<UserInfoProps> = ({
                         <Tooltip
                           className="flex min-w-[4.5rem] items-center justify-end text-sm text-blue-400"
                           tooltipContent={
-                            <Numeral value={reputationRaw} suffix=" pts" />
+                            <Numeral
+                              value={reputationRaw}
+                              suffix=" pts"
+                              decimals={getTokenDecimalsWithFallback(
+                                colony.nativeToken.decimals,
+                                DEFAULT_TOKEN_DECIMALS,
+                              )}
```

Comment:
Given the `getTokenDecimalsWithFallback` already uses `DEFAULT_TOKEN_DECIMALS` as it's fallback value I think we can skip passing it as the 2nd argument

Reply:
Oh you're right @mmioana , you know what I'll clean up other places where this is being done as well 👌

Reply:
@mmioana pushed! 🚀  I've also cleaned up other areas that use this redundant 2nd argument